packages/ui/cypress/support/Extension.ts

```diff
@@ -24,7 +24,7 @@ export type TxRequests = Record<number, TxRequest>
 export type AuthRequests = Record<number, AuthRequest>
 
 export type EnableRequest = number
-
+const ss58Format = 2
```

Comment:
It probably should be KUSAMA_S58_FORMAT

Reply:
Agreed. Actually I removed it since it's not needed.